### ISSUE TITLE
Git Releases/Versioning

### ISSUE THREAD

**Issue by GoonerWare** [araten10/EdgewarePlusPlus#123]
Hi there,

I can't recall if there were ever releases for this repo and if they've just disappeared for some reason, but if not I was wondering if it'd be possible to have the finalised versions (e.g. 14.2 yesterday) tagged?

It makes looking at the file changes between versions much easier than having to find specific git IDs!

Thanks so much!

**Comment by LewdDevelopment** [comment]
I've talked about using GitHub releases before with @araten10 but I guess they forgot about it and I never checked if I actually have permissions to do that, oops. Better late than never!

**Comment by araten10** [comment]
yes, I apologize. This project is the first time I have used github in a semi-professional manner, and marigold has mentioned releases to me multiple times but I kept forgetting or putting them off. I see no problem with starting to do them, I just also don't understand them much and haven't habitified them yet

**Comment by GoonerWare** [comment]
Thanks very much both, I really appreciate it :)
